ptt.py

```diff
@@ -0,0 +1,10 @@
+import pyttsx3
+
+engine = pyttsx3.init()
+voices = engine.getProperty("voices")
```

Comment:
What is this file for

Reply:
This file is for someone to check the index for python_voice. So it's optional, but it will make it easier for the user to use pyttsx